### PR TITLE
Add contributors

### PR DESCRIPTION
**Pull request by jonathanlei**
add a thank you for contributing section to recognize all contributors so far.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,19 @@ If you see a bounty that you think is particularly important you can upvote it t
 
 <br>
 
+# Thanks For Contributing
+XRPL Bounties would like to express our sincere gratitude to all of the contributors who have helped to improve the XRPL ecosystem through their participation in bounties. Your hard work and dedication are highly valued and appreicated. 
+## Contributor list (No Particular Order):
+- [kasiimh1](https://github.com/kasiimh1)
```
Comment (mvadari) [inline README.md:117]: What about just putting it in alphabetical order by username?

Comment (wojake) [reply README.md:117]: Agreed and the list shouldn't be based on the number of milestones and what not, just alphabetical order to show gratitude to all contributors 👍🏼

**Review by mvadari — APPROVED** [review]

**Review by ckniffen — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -111,6 +111,20 @@ If you see a bounty that you think is particularly important you can upvote it t
 
 <br>
 
+# Thanks For Contributing
+XRPL Bounties would like to express our sincere gratitude to all of the contributors who have helped to improve the XRPL ecosystem through their participation in bounties. Your hard work and dedication are highly valued and appreicated.
```
Comment (intelliot) [inline README.md:115]:
```suggestion
XRPL Bounties expresses sincere gratitude to all of the contributors who help to improve the XRP Ledger and related projects. Your hard work and dedication are highly valued and appreciated. 
```

File: README.md [hunk 3]
```diff
@@ -111,6 +111,20 @@ If you see a bounty that you think is particularly important you can upvote it t
 
 <br>
 
+# Thanks For Contributing
+XRPL Bounties would like to express our sincere gratitude to all of the contributors who have helped to improve the XRPL ecosystem through their participation in bounties. Your hard work and dedication are highly valued and appreicated. 
+## Contributor list (In Alphabetical Order):
```
Comment (intelliot) [inline README.md:116]:
```suggestion
## Contributor List (In Alphabetical Order):
```

**Review by khancode — APPROVED** [review]
LGTM once you address all comments

**Pull request merged**